### ISSUE TITLE
Images not loaded in localhost

### ISSUE THREAD

**Issue by gharach** [noumo/easyii#182]
after I installed easyiicms in localhost image did'nt show in any pages!
![image](https://cloud.githubusercontent.com/assets/5032193/18417757/051ac97c-784e-11e6-95f9-bcac009ea65b.png)

but when I modified core image helper of yii framework in the BaseHtml.php file  with these line all image loaded

`public static function img($src, $options = [])
    {
        $options['src'] = Url::toRoate($src);
        if (!isset($options['alt'])) {
            $options['alt'] = '';
        }
        return static::tag('img', '', $options);
    }`

as you can see I change first line of function img  and replace Url:to($scr) with Url::toRoate
and images load correctly then .
as changing yii core is not a good solution what should I do ??

**Comment by mkrvevgn** [comment]
This problem is already discribed and solution already exists

https://github.com/noumo/easyii/issues/133

**Comment by gharach** [comment]
yes , but I dont wanna change httpd.conf file , what should I do now?

**Comment by JDL747** [comment]
It didn't worked even after changing the httpd.conf file !

The workaround is to change line 14 in vendor/noumo/easyii/helpers/Upload.php like so

`public static $UPLOADS_DIR = '../{projectName}/uploads';`

**Comment by findinglx** [comment]
> @JDL747  Thank U

**Comment by barakaPBM** [comment]
@JDL747  This worked for me Thank you

**Comment by JDL747** [comment]
@barakaPBM this CMS seems not to be maintained anymore, I recommend this one https://luya.io/

**Comment by dzyof** [comment]
`$asset = \app\assets\AppAsset::register($this);`
`Html::img($asset->baseUrl . '/images/pic_logo_ua.png', ['alt' => 'logo'])`

**Comment by izomi32** [comment]
thx ^_^
